_data/documentation/latest.yaml

```diff
@@ -0,0 +1,37 @@
+- Name: Intro
+  Childs:
+  - Name: What is flotta
+    Link: "intro/overview.md"
+  - Name: Use cases
+    Link: "intro/usecases.md"
+  - Name: Flotta vs others
+    Link: "intro/vs.md"
+
+- Name: Getting started
+  Childs:
+  - Name: Minikube
+    Link: "gsg/minikube.md"
+  - Name: Kind
+    Link: "gsg/kind.md"
+  - Name: OCP
+    Link: "gsg/ocp.md"
+
+- Name: Operations
+  Childs:
+  - Name: System Requirements
+    Link: "operations/requirements.md"
+  - Name: Configuration
+    Link: "operations/configuration.md"
+  - Name: Observability
+    Link: "operations/observability.md"
+  - Name: Data upload
+    Link: "operations/data_upload.md"
+  - Name: Troubleshooting
+    Link: "operations/troubleshooting.md"
+  - Name: API reference
+    Link: "operations/api.html"
```

Comment:
These files are downloaded and not opened in the browser. Is this a known behavior?

Reply:
Because are not yet created, it'll be fixed when guides are in place.